packages/cli/src/commands/bundle.ts

```diff
@@ -61,23 +63,26 @@ export const bundleAction = ({
       componentExtractStep.success(getComponentCountText(componentCount, fileCount));
 
       const decoratorFiles = await glob(decoratorPath, { cwd: executionPath });
+      const storyFiles = await glob(storyPath, { cwd: executionPath });
       const decoratorFileArray =
         decoratorFiles.length > 0 ? [pathUtils.resolve(executionPath, decoratorFiles[0])] : [];
+      const storyFileArray = storyFiles.map(sf => pathUtils.resolve(executionPath, sf));
 
-      const entrypoints = Object.entries(entrypointsWithMetadata).reduce(
-        (prev, [key, ep]) => ({
+      const entrypoints = Object.entries(entrypointsWithMetadata).reduce((prev, [key, ep]) => {
+        const abc = storyFileArray.filter(sf => sf.includes(key));
```

Comment:
abc? can we rename this to sth that means sth?

Reply:
Ah. damn. That logic is even wrong :D

Reply:
done